files/qsub.pl

```diff
@@ -583,6 +587,29 @@ sub parse_script
         }
     };
 
+    # handle queues
+    if ($orig_argsh{'q'}) {
```

Comment:
i would warn the same message for both arg and/or directive

Reply:
Done.